web/elm/src/Dashboard/Group.elm

```diff
@@ -374,3 +434,28 @@ pipelineDropAreaView dragState name { x, y, width, height } target =
         , stopPropagationOn "drop" (Json.Decode.succeed ( DragEnd, True ))
         ]
         []
+
+
+headerView : PipelineGrid.Bounds -> String -> Html Message
+headerView { x, y, width, height } header =
```

Comment:
good header styles, i tried fooling it with some super long team names but it bested me!

Reply:
The machines are outsmarting us all!